src/index.html

```diff
@@ -7,8 +7,56 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <!-- Header -->
+    <header class="container">
+      <a href="#home" class="moyo-logo">
+        <img src="images/logo.png" alt="moyo-logo">
+      </a>
+      <!-- Navigation -->
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
add an empty line between _multiline_ siblings for better readability
```suggestion
          </li>
          
          <li class="nav__item">
```